programmatic-upload-gcp-cli.md

```diff
@@ -33,3 +33,4 @@ Once your data is in the bucket, it will automatically be synced with your cente
 **Note**: If you would like to make changes to your data, please do so directly from the S3 bucket and not from the Synapse web or programmatic clients. Changes made to the bucket will automatically be updated on the Synapse project. 
```

Comment:
Vesteinn caught a typo here- change 'S3 bucket' to 'Google Storage Bucket'

Reply:
Ooo good catch! updated.

Reply:
Thanks Yooree for updating! (don't think I see a new commit with these changes though?)

Vesteinn also [mentioned](https://github.com/Sage-Bionetworks/HTAN-coordination/issues/170#issuecomment-702798667) we should specify that the email addresses required for Google bucket access must be Gmail/G suite emails

Reply:
ah, just pushed the commits! I'll add in the gsuite comments next